ui/src/router/components/form/steps/EnsemblerStep.js

```diff
@@ -69,7 +69,7 @@ export const EnsemblerStep = ({ projectId }) => {
         />
       )}
 
-      {ensembler.type === "standard" && (
+      {experiment_engine.type !== "nop" && ensembler.type === "standard" && (
```

Comment:
Instead of doing this, can we handle this in `onChangeEngineType` in `ui/src/router/components/form/steps/ExperimentStep.js`? You are currently clearing the standard ensembler properties if the engine type is changed. In addition, can we handle enginetype becoming none?

```javascript
const onChangeEngineType = (newType) => {
    onChange("config.experiment_engine")({ type: newType });
    if (ensemblerType === "standard") {
      // Reset standard ensembler configs
      if (newType === "nop") {
           onChange("config.ensembler.type")("nop");
      }
      onChange("config.ensembler.standard_config.experiment_mappings")([]);
      onChange("config.ensembler.standard_config.route_name_path")("");
    }
}
```

Reply:
Oh that's true that makes more sense 😅 I'll make the change and then test it out again before merging this! Thanks!